exceptions/part-2/assets/excerpts.md

```diff
@@ -0,0 +1,73 @@
+# Excerpts
+
+You'll find below excerpts from the following texts:
+
+* 1993 : McConnell, Code complete;
+* 2008 : Robert Matin, Clean code;
+* 2018 : Douglas Crockford, How Javascript works.
+
```

Comment:
Ajouter Implementations Patterns (Beck) et The Pragmatic Programmer (Hunt and Thomas)